### PR TITLE
Add page for Team

### PR DESCRIPTION
**Pull request by asgrim**
Just an idea I had. No idea what it's like on mobile because I suck at doing front end stuff :)

### CODE REVIEW

**Comment by lboynton** [comment]
Twitter links?

File: _sass/pages/_team.scss [hunk 1]
```diff
@@ -0,0 +1,26 @@
+.team-member-card {
+    width: 18%;
+    border: 1px solid silver;
+    -webkit-border-radius: 10px;
+    -moz-border-radius: 10px;
```
Comment (lboynton) [inline _sass/pages/_team.scss]: These vendor prefixed versions are quite old now, they aren't needed anymore

File: team.html [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+            bookings, signing contracts, ordering stationary, herding speakers, and keeping the website running, these
+            folks are core organisation team.
+        </p>
+
+        {% for person in page.team %}
+        <div class="team-member-card">
+            <h3>{{ person.who }}</h3>
+            <h4>{{ person.role }}</h4>
+            <img src="/img/team/{{ person.avatar }}" class="img-responsive" alt="{{ person.who }}">
+        </div>
```
Comment (lboynton) [inline team.html:62]: To make this responsive, you could try:

```
<div class="row">
    <div class="col-md-2 col-md-offset-1">person1</div>
    <div class="col-md-2">person2</div>
    <div class="col-md-2">person3</div>
    <div class="col-md-2">person4</div>
    <div class="col-md-2">person5</div>
</div>
```

File: team.html [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+team:
+- who: James Titcumb
+  avatar: james.png
+  role: Conference Chair
+- who: Alex Ross
+  avatar: alex.jpg
+  role: Materials, print, design
+- who: Gary Hockin
+  avatar: gary.jpg
+  role: Speaker
```
Comment (lboynton) [inline team.html]: Should this be "Speaker liaison"?

File: team.html [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+<div id="main" class="team">
+    <div class="container">
+        <header>
+            <h2>Organisation Team</h2>
+        </header>
+
+        <p>
+            The organisation team are responsible for ensuring the smooth running of the conference. From making
+            bookings, signing contracts, ordering stationary, herding speakers, and keeping the website running, these
+            folks are core organisation team.
```
Comment (lboynton) [inline team.html]: are _part of the_ core organisation team

**Review by lboynton — CHANGES_REQUESTED** [review]

**Comment by asgrim** [comment]
@lboynton updated I think :D

**Comment by lboynton** [comment]
@asgrim can I update this PR?

**Comment by asgrim** [comment]
@lboynton yes please! :)

**Comment by lboynton** [comment]
How about something like this?

phpsc17-team

**Comment by asgrim** [comment]
Yeah that's better :D hehe

**Comment by asgrim** [comment]
Is this ready to merge btw?

**Comment by lboynton** [comment]
No I didn't finish making changes :stuck_out_tongue_closed_eyes: 

Sorry I am on this tomorrow!

**Review by lboynton — APPROVED** [review]

**Pull request merged**